src/server/Main.cpp

```diff
@@ -112,9 +114,9 @@ int main() {
             [&server](const ECS::Entity &e) { server.delete_entity(e); }
         );
 
-        // engine.getRegistry().componentManager().registerGlobalCreateCallback(
-        //     [&server](std::type_index type, size_t index) { server.attach_component(index, type); }
-        // );
+        engine.getRegistry().componentManager().registerGlobalCreateCallback(
```

Comment:
Could you explain why you are uncommenting this part ? :) We had trouble in the past with async sending infos not at the right time that lead us to comment this, does it work now ?

Reply:
I did so after a merge, and given that the incoming changes were not commented i assumed it was okay to uncomment this line too, i wasn't aware it had to stay commented. It has been commented again as i do not know if it works now.

Reply:
Okay !